Helper/Cart.php

```diff
@@ -684,12 +684,11 @@ public function convertCustomAddressFieldsToCacheIdentifier($quote)
     protected function getImmutableQuoteIdFromBoltOrder($boltOrder)
     {
         $response = $boltOrder ? $boltOrder->getResponse() : null;
-        list(, $immutableQuoteId) = $response ? explode(' / ', $response->cart->display_id) : [null, null];
-        return $immutableQuoteId;
+        return $response->cart->metadata->immutable_quote_id;
```

Comment:
I can imagine that immutable_quote_id doesn't exist. For example, bolt order created by previous plugin version and we handle it right after update. Could we add bugsnag here for now and maybe think about this case in next PRs?

Reply:
Fixed. I also missed the fact that `$response` could be `null` as defined by the very flow here.